### PR TITLE
Adding Database jokes

### PR DESCRIPTION
**Pull request by goellavish10**
There are very few database jokes in the list, just added few to make the package more interesting

### CODE REVIEW

File: puns.json [hunk 1]
```diff
@@ -438,5 +438,13 @@
   {
     "pun": "Q: Why are javascript interviews so unpredictable?",
     "punchline": "A: You never know how a candidate will React!"
+  },
+  {
+    "pun":"I saw a great movie about databases today",
```
Comment (AlexLakatos) [inline puns.json:443]:
```suggestion
    "pun": "Q: Why are there not great movies about databases?",
```

File: puns.json [hunk 2]
```diff
@@ -438,5 +438,13 @@
...
+    "punchline":"I can't wait for the SQL"
```
Comment (AlexLakatos) [inline puns.json:444]:
```suggestion
    "punchline": "Because everyone already hates the SQL!"
```

File: puns.json [hunk 3]
```diff
@@ -438,5 +438,13 @@
...
+  },
+  {
```
Comment (AlexLakatos) [inline puns.json:446]: There's a few puns about joining the tables already, can you remove this one?

**Review by AlexLakatos — CHANGES_REQUESTED** [review]
I'm trying to keep these as a "questions and answers" type deal, it works better with a pause for Voice apps. I made some suggestions, let me know if that's OK.

File: puns.json (hunk 3 above)
Comment (goellavish10) [reply puns.json:446]: Sure

**Comment by AlexLakatos** [comment]
Merged #101 instead.

**Pull request closed without merging**